config/tool_conf.xml.sample

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<toolbox>
+<toolbox monitor="true">
```

Comment:
My tool_conf.xml.sample was unchanged when I pulled these changes, some gitignore (that has `config/*`) thing?